scripts/plot_improvement.py

```diff
@@ -59,7 +60,7 @@ def mean_makespan(bench_obj):
 
     x, y = mean_makespan(bench_obj)
     plt.plot(x, y, marker=".", label="Mean makespan")
-
+    print(iterations)
```

Comment:
This should be removed
```suggestion
```

Reply:
Of course, it should just be an example of how to do something with the iterations